ros2controlcli/doc/userdoc.rst

```diff
@@ -74,6 +75,45 @@ Example output:
     joint_trajectory_controller/JointTrajectoryController                  controller_interface::ControllerInterface
 
 
+list_hardware_components
+------------------------
+
+.. code-block:: console
+
+    $ ros2 control list_hardware_components -h
+    usage: ros2 control list_hardware_components [-h] [--spin-time SPIN_TIME] [-s] [-c CONTROLLER_MANAGER] [--include-hidden-nodes]
+
+    Output the list of available hardware components
+
+    options:
+    -h, --help            show this help message and exit
+    --spin-time SPIN_TIME
```

Comment:
Do we need this parameter?

Reply:
@bmagyar I actually don't understand how `--spin-time` is added as an argument. Is this through [ros2cli/ros2cli/node/direct.py](https://github.com/ros2/ros2cli/blob/f46fa80c5bffe95b45fbba2e5cda724f3e4e8cce/ros2cli/ros2cli/node/direct.py)? Can you specify you don't need this option?

Reply:
I think you are right, we probably can't...